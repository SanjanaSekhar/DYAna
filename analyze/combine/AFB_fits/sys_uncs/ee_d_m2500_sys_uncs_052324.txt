Systematic uncertainties for bin 0
LQ LO reweighting & 0.08012 & 44.51  \\ 
Electron Momentum Scale & 0.05687 & 22.42  \\ 
Electron Identification/Isolation & 0.05375 & 20.03  \\ 
$\alpha_s$ + Renormalization/Factorization Scales & 0.02797 & 5.42  \\ 
DY Cross Section & 0.02623 & 4.77  \\ 
$\ttbar$ Cross Section & 0.02022 & 2.83  \\ 
Total Uncertainty & 0.12  \\ 
