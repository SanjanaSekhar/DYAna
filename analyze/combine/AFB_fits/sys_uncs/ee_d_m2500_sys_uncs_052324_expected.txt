Systematic uncertainties for bin 0
LQ LO reweighting & 0.44366 & 39.74  \\ 
Electron Identification/Isolation & 0.37675 & 28.66  \\ 
$\alpha_s$ + Renormalization/Factorization Scales & 0.26489 & 14.17  \\ 
$\ttbar$ Cross Section & 0.25636 & 13.27  \\ 
Electron Momentum Scale & 0.11534 & 2.69  \\ 
DY Cross Section & 0.08553 & 1.48  \\ 
Total Uncertainty & 0.70  \\ 
